scipy/cluster/vq.py

```diff
@@ -519,7 +529,49 @@ def _krandinit(data, k):
     return x
 
 
-_valid_init_meth = {'random': _krandinit, 'points': _kpoints}
+def _kpp(data, k):
+    """ Picks k points in data based on the kmeans++ method
+
+    Parameters
+    ----------
+    data : ndarray
+        Expect a rank 1 or 2 array. Rank 1 are assumed to describe one
+        dimensional data, rank 2 multidimensional data, in which case one
+        row is one observation.
+    k : int
+        Number of samples to generate.
+
+    Returns
+    -------
+    init : ndarray
+        A 'k' by 'N' containing the initial centroids
+
+    References
+    ----------
+    D. Arthur and S. Vassilvitskii, "k-means++: the advantages of careful seeding",
+      Proceedings of the Eighteenth Annual ACM-SIAM Symposium on Discrete Algorithms, 2007.
```

Comment:
This causes a rendering issue (from CI: `(WARNING/2) Enumerated list ends without a blank line; unexpected unindent.::`).

See section 13 of https://numpydoc.readthedocs.io/en/latest/format.html#sections for the correct formatting